Add documentation for rate limiting rules (#43)
ONSdigital/dp-developer-site: static/index.md

[Guide to filtering a dataset](filters/)

### Rate Limiting Rules

> The following rate limiting rules have been implemented on the API

Reviewer: Few lines to explain why we are adding the rate limit would be good. something along the lines of "We limit the number of requests you can make to dp API's to protect our service against excessive use and denial of service attacks ."

Reviewer: We also need to mention about the  429 response when we exceed the limit stated and about the Retry-header as mentioned in the trello ticket

Reviewer: And can you mention both the domains as well

[Guide to filtering a dataset](filters/)

> ### Rate Limiting Rules

Reviewer: Looking at other subheading , this should be `Rate limiting rules`. L and R (lowercase)

> Rate limiting rules have been implemented to limit the number of requests a client can make to The Office for National Statistics API. This is to protect the service against excessive use and denial of service attacks. 

Reviewer 2: ```suggestion
Rate limiting has been implemented to contain the number of requests a client can make to our API. This is to protect the service against excessive use and denial of service attacks. 
```

1. repeated 'limit' was repetitive 😄 
2. removing 'rules' sounds more direct

> The following rate limiting rules have been implemented on the API. 

Reviewer 2: ```suggestion
The following rate limits have been implemented on the API:
```

* 120 requests per 10 seconds

* 200 requests per 1 minute

> They have been applied to following domains. 

Reviewer 2: ```suggestion
They have been applied to following domains:
```

* api.ons.gov.uk

* api.beta.ons.gov.uk

> If requests are made that reach these limits the IP address of the client will be blocked for 1 minute. The client will receive a JSON response with a 429 error code, the retry-after header shows the seconds remaining until a retry can be made.  

Reviewer: The client will receive a JSON response with a 429 error code. The Retry-After header shows the seconds remaining until a retry can be made.

> If requests are made that reach these limits the IP address of the client will be blocked for 1 minute. The client will receive a 429 response, the retry-header contains the seconds remaining until a retry can be made.  

Reviewer: the retry-header `shows` or something similar would make sense . `retry-header contains the seconds' seems out of place .
